crates/driver/src/domain/competition/auction.rs

```diff
@@ -112,6 +113,30 @@ impl Auction {
     pub fn score_cap(&self) -> Score {
         self.score_cap
     }
+
+    /// All auction prices normalized to native token price.
+    pub fn normalized_prices(&self) -> NormalizedPrices {
```

Comment:
Can we not introduce a new type here? I'm a bit confused as to what the normalisation aspect refers to (normalizing into native token or normalizing with respect to converting wei amounts into unit amounts).

Auction already defines per token prices and I think we can just for convenience return a `Prices = HashMap<address, Price>`. If the callsite requires `BigRationals` for precision that's fine but should stay local to the callsite (auction itself doesn't do fractions).

I think it's ok to add the buy eth address for convenience (but I'd document this in the comment)

Reply:
Fixed.